tests/e2e/logging/test_openshift-logging.py

```diff
@@ -140,3 +141,70 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
             logger.info("The new project exists in the EFK stack")
         else:
             raise ModuleNotFoundError
+
+    @retry(ModuleNotFoundError, 6, 300, 3)
+    @pytest.mark.polarion_id("OCS-651")
+    @tier4
+    def test_respin_elasticsearch_pod(self, create_pvc_and_deploymentconfig_pod):
+        """
+         Test to verify respin of elasticsearch pod has no functional impact
```

Comment:
Please align text with the quotation marks

Reply:
Done